2.0/m_antirandom.cpp

```diff
@@ -550,9 +567,10 @@ class ModuleAntiRandom : public Module
 		unsigned int digits = 0;
 
 		/* Fast digit/consonant/vowel checks... */
-		for (s = original_str; *s; s++)
+		for (i = 0; i < original_str.length(); i++)
```

Comment:
If `i` is only used in the loop restrict the scope of it to the loop by declaring it in the init section of the `for`